src/estimators/bloom_filters.py

```diff
@@ -143,6 +143,33 @@ def __init__(self, length, random_seed=None):
         random_seed)
 
 
+class GeometricBloomFilter(AnyDistributionBloomFilter):
+  """Implement a Geometric Bloom Filter."""
+
+  @classmethod
+  def get_sketch_factory(cls, length, num_hashes=1):
```

Comment:
Do you need multiple hashes? Line 114 will assert on num_hashes > 1.

Reply:
Removed